src/preamble.js

```diff
@@ -948,18 +948,18 @@ function createWasm() {
   // performing other necessary setup
   /** @param {WebAssembly.Module=} module*/
   function receiveInstance(instance, module) {
-    var exports = instance.exports;
+    var wasmExports = instance.exports;
```

Comment:
I think you want to drop the `var ` here, since `wasmExports` is a global

Reply:
Sure, its a slightly more invasive/risky change, but its also better since its saves on code size.

Assuming the tests pass I think its very safe.

Reply:
Done